addons/mail/static/src/models/thread/thread.js

```diff
@@ -831,6 +839,17 @@ function factory(dependencies) {
             return this.messaging.currentPartner.moderatedChannelIds.includes(this.id);
         }
 
+        /**
+         * @private
+         * @returns {boolean}
+         */
+        _computeIsPinned() {
+            if (this.group_based_subscription) {
```

Comment:
I have absolutely no idea that this feature exists or what it does.

Reply:
Add comment in code to explain current need/issue + add test to cover it (group_based_subscription with is_pinned = false)